demos/multi_channel_common/cpp/multichannel_params.hpp

```diff
@@ -13,8 +13,8 @@ static const char input_message[] = "Required. A comma separated list of inputs
     "single image, a folder of images or anything that cv::VideoCapture can process.";
 static const char loop_message[] = "Optional. Enable reading the inputs in a loop.";
 static const char duplication_channel_number_message[] = "Optional. Multiply the inputs by the given factor. For "
-    "example, if only one input is provided, but -ni is set to 2, the demo uses half of images from the input as it was"
-    " the first input and another half goes as the second input.";
+    "example, if only one input is provided, but -duplicate_num is set to 2, the demo uses real input as the first input channel"
+    " and duplicated the same data as the second input channel.";
```

Comment:
It doesn't truly duplicate data. It splits it. One image from input will go to only one channel. You should revert everything but `-ni`->`-duplicate_num` change.

Reply:
could you please explain, so it reduces number of frames at each channel and introduce latency at each channel?

Reply:
Yes, each channel gets less frames. Each frame is forwarded to one of channels, but it doesn't introduce extra latency. What does introduce latency is a buffer used to connect a thread which performs frame reading and a thread performing preprocessing and scheduling inference.

Reply:
in case 4 duplicated channels, each channel get 4-th frame from video file, and have to wait until next 4-th frame decoded?

Reply:
Correct. But the latency as it is defined for object_detection_demo wouldn't be affected if it was implemented here. It decreases frame rate of a channel

Reply:
well, need to think how to describe this better

Reply:
corrected `duplicate_num` description